Cargo.toml

```diff
@@ -26,26 +24,26 @@ serde_json = { version = "1.0", optional = true }
 ordered-float = "2.0"
 warp10 = { version = "2.0.0", optional = true }
 rand = { version = "0.7.3" }
-time = "0.3.20"
-colored = "2.0.0"
-chrono = "0.4.19"
+time = "0.3"
+colored = "2.0"
+chrono = "0.4"
 docker-sync = { version = "0.1.2", optional = true }
 k8s-sync = { version = "0.2.3", optional = true }
 hyper = { version = "0.14", features = ["full"], optional = true }
 tokio = { version = "1.26.0", features = ["full"], optional = true}
 sysinfo = { version = "0.28.3"}
 
 [target.'cfg(target_os="linux")'.dependencies]
-#procfs = "0.13.2"
-procfs = { version = "0.12.0" }
+procfs = { version = "0.15.0" }
 
 [target.'cfg(target_os="windows")'.dependencies]
 windows = { version = "0.27.0", features = ["alloc","Win32_Storage_FileSystem","Win32_Foundation","Win32_Security","Win32_System_IO","Win32_System_Ioctl"]}
 
 [features]
-default = ["prometheus", "riemann", "warpten", "json", "containers"]
+default = ["prometheus", "riemann", "warpten", "json", "containers", "qemu"]
```

Comment:
is there an issue to not include qemu as a default feature ?

Reply:
It doesn't, so I think we shouldn't include it by default.

I was wondering if you faced an issue because of this absence.